net/strongswan/files/swanctl.init

```diff
@@ -654,6 +654,8 @@ config_ipsec() {
 }
 
 do_postamble() {
+	swanctl_xappend0 "include /var/swanctl/conf.d/*.conf"
```

Comment:
Why before the files in `/etc` and not after?

Reply:
swanctl_xappend0 manipulates `/var/swanctl/swanctl.conf`, lines down below modifies `/var/ipsec/strongswan.conf`, I'm not sure I understand what files `/etc` are you referring to?